src/settings/Settings.js

```diff
@@ -174,12 +174,12 @@ export const SETTINGS = {
     "fontSizeMin": {
         displayName: _td("Min font size"),
         supportedLevels: LEVELS_ACCOUNT_SETTINGS,
-        default: 14,
+        default: 13,
     },
     "fontSizeMax": {
         displayName: _td("Max font size"),
         supportedLevels: LEVELS_ACCOUNT_SETTINGS,
-        default: 24,
+        default: 20,
     },
```

Comment:
It looks like these settings aren't something the user can change and are instead hardcoded to a particular value. A pair of constants somewhere in the code would be preferred to avoid abusing the settings system (and to avoid a situation when someone inevitably tries to set this to `min: 10, max: -10` with localstorage).